libs/components/packages/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/packages",
-  "version": "0.0.0-PACKAGES_PLACEHOLDER",
```

Comment:
any particular reason we're removing the "packages" part of the version?

Reply:
The `PACKAGES` part of the variable was used during the build step for SKY UX 5 when our `@skyux/packages` version was different from the other packages in the monorepo. Now, everything uses the same version.